!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  blue_mask_h_max: 115
  blue_mask_h_min: 96
  blue_mask_px_max: 720
  blue_mask_px_min: 0
  blue_mask_py_max: 1280
  blue_mask_py_min: 0
  blue_mask_s_max: 255
  blue_mask_s_min: 207
  blue_mask_v_max: 255
  blue_mask_v_min: 177
  board_blue_mask_h_max: 120
  board_blue_mask_h_min: 70
  board_blue_mask_s_max: 255
  board_blue_mask_s_min: 200
  board_blue_mask_v_max: 255
  board_blue_mask_v_min: 110
  board_mask_h_max: 179
  board_mask_h_min: 43
  board_mask_s_max: 255
  board_mask_s_min: 69
  board_mask_v_max: 255
  board_mask_v_min: 20
  debug_img_to_show: -1
  dilate_iterations: 4
  erode_iterations: 4
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      blue_mask_h_max: 115
      blue_mask_h_min: 96
      blue_mask_px_max: 720
      blue_mask_px_min: 0
      blue_mask_py_max: 1280
      blue_mask_py_min: 0
      blue_mask_s_max: 255
      blue_mask_s_min: 207
      blue_mask_v_max: 255
      blue_mask_v_min: 177
      board_blue_mask_h_max: 120
      board_blue_mask_h_min: 70
      board_blue_mask_s_max: 255
      board_blue_mask_s_min: 200
      board_blue_mask_v_max: 255
      board_blue_mask_v_min: 110
      board_mask_h_max: 179
      board_mask_h_min: 43
      board_mask_s_max: 255
      board_mask_s_min: 69
      board_mask_v_max: 255
      board_mask_v_min: 20
      debug_img_to_show: -1
      dilate_iterations: 4
      erode_iterations: 4
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      px_max: 720
      px_min: 0
      py_max: 1022
      py_min: 111
      red_dilate_iterations: 1
      red_erode_iterations: 1
      red_mask1_h_max: 18
      red_mask1_h_min: 0
      red_mask1_s_max: 255
      red_mask1_s_min: 100
      red_mask1_v_max: 255
      red_mask1_v_min: 20
      red_mask2_h_max: 179
      red_mask2_h_min: 165
      red_mask2_s_max: 255
      red_mask2_s_min: 100
      red_mask2_v_max: 255
      red_mask2_v_min: 20
      red_mask_px_max: 499
      red_mask_px_min: 241
      red_mask_py_max: 792
      red_mask_py_min: 434
      state: true
      type: ''
      visual_servoing_linear_gain: 0.0003
      visual_servoing_linear_saturation: 0.0001
      visual_servoing_position_error_tolerance: 0.0001
    state: []
  px_max: 720
  px_min: 0
  py_max: 1022
  py_min: 111
  red_dilate_iterations: 1
  red_erode_iterations: 1
  red_mask1_h_max: 18
  red_mask1_h_min: 0
  red_mask1_s_max: 255
  red_mask1_s_min: 100
  red_mask1_v_max: 255
  red_mask1_v_min: 20
  red_mask2_h_max: 179
  red_mask2_h_min: 165
  red_mask2_s_max: 255
  red_mask2_s_min: 100
  red_mask2_v_max: 255
  red_mask2_v_min: 20
  red_mask_px_max: 499
  red_mask_px_min: 241
  red_mask_py_max: 792
  red_mask_py_min: 434
  visual_servoing_linear_gain: 0.0003
  visual_servoing_linear_saturation: 0.0001
  visual_servoing_position_error_tolerance: 0.001
state: []
